lib/croods/resource/sorting.rb

```diff
@@ -3,10 +3,26 @@
 module Croods
   module Resource
     module Sorting
+      def order_by_attribute
+        @order_by_attribute ||= Croods::Attribute.new(
+          'order_by', :string, null: true
+        )
+      end
+
+      def order_attribute
+        @order_attribute ||= Croods::Attribute.new(
+          'order', :string, null: true
+        )
+      end
+
+      def order_params
+        @order_params ||= [order_by_attribute, order_attribute]
+      end
+
       def sort_by(attribute = nil)
```

Comment:
Technically it's not always an attribute anymore, it's a symbol that could be a method too.